src/esp/bindings/AttributesManagersBindings.cpp

```diff
@@ -237,6 +239,15 @@ void declareBaseAttributesManager(py::module& m,
             "if it does not.")
                .c_str(),
            "handle"_a)
+      .def("get_first_matching_template_by_handle",
```

Comment:
Why introduce this instead of getting all matches and selecting the first like we do already? Doing that puts responsibility on user code to handle the case of multiple matches and offers the opportunity for an assertion.

Reply:
I was doing this to provide a shortcut for folks who know they are going to use the first handle they get back, so they didn't have to deal with querying the handles and then querying for the actual attributes.  Seems like the first index tends to be the goto for this kind of process, and I thought I'd abstract it away. The other mechanism is still present.